docs/conf.py

```diff
@@ -75,7 +75,8 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_nefertiti"
+# html_theme = "sphinx_nefertiti"
```

Comment:
```suggestion
```

No need to leave this comment :)

Reply:
Booooooooooooooooooooo. I am having regret because the current one is so beautiful